OpenRA.Mods.Common/UpdateRules/Rules/TextNotificationsDisplayWidgetRemoveTime.cs

```diff
@@ -0,0 +1,39 @@
+﻿#region Copyright & License Information
```

Comment:
This needs to be placed in a new folder. It also isn't mentioned in the UpdatePath